docs/_docs/Developers/build.md

```diff
@@ -5,45 +5,91 @@ order: 1
 toc: true
 ---
 
-## Ubuntu 18.04/18.10 and Linux Mint 19
+## Ubuntu 18.04/18.10
+*Updated by [kreyren](https://github.com/Kreyren) at 24.02.2019*
 
-* Install the dependencies:
-```
+Fork https://github.com/PhoenicisOrg/phoenicis on github before building.
+
+#### Dependencies:
+- git
```

Comment:
Why? This means that we always have to update the dependencies twice.

Reply:
@plata elaborate?

Reply:
The dependency is listed in "Dependencies" and in the `apt install` line.

Reply:
@plata Listed as information and then listed in command in how to get those dependencies not sane?

Reply:
In theory we can put dependencies in separate block, but they might be different for each distro

Reply:
In my opinion, it's sufficient to have the command (what would you want to do with the dependencies except of installing them?).

Reply:
@plata it's on the same line 
![image](https://user-images.githubusercontent.com/11302521/53590037-2b0bc880-3b91-11e9-9ac0-8c37f8f36a4c.png)

There are no duplicates of the same commands

Reply:
Not the commands, e.g. fakeroot is listed as dependency and also in the apt command. So if something changes, it must be changed in two places. That's not good.